data/Templates/STU3R4/metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+	"type": "json"
```

Comment:
If we define the type of these template as `json`, do we need to consider changing the directory structure of templates from `Templates/STU3R4` to `Templates/Json/STU3R4` ?  The same goes for data from `SampleData/Stu3` to `SampleData/Json/Stu3`.